pkg/providers/amifamily/al2.go

```diff
@@ -63,13 +65,14 @@ func (a AL2) DescribeImageQuery(ctx context.Context, ssmProvider ssm.Provider, k
 			IsMutable: amiVersion == v1.AliasVersionLatest,
 		})
 		if err != nil {
+			ssmErr = err
```

Comment:
I think we should consider using the same `multierr.Append` pattern that we have elsewhere throughout the project and then do an error check directly after L72